26/upgrade.html

```diff
@@ -19,6 +19,48 @@
 
 <script id="upgrade-template" type="text/x-handlebars-template">
 
+<h4><a id="upgrade_2_6_0" href="#upgrade_2_6_0">Upgrading from 0.8.x, 0.9.x, 0.10.0.x, 0.10.1.x, 0.10.2.x, 0.11.0.x, 1.0.x, 1.1.x, 2.0.x or 2.1.x or 2.2.x, 2.3.x, 2.4.x, 2.5.x to 2.6.0</a></h4>
```

Comment:
This got pretty unwieldy. Can we simplify it somehow?

Reply:
Ack.